examples/go/cmd/demo-audio/main.go

```diff
@@ -198,8 +205,7 @@ func (manager *sessionManager) ToggleOff() {
 // StartEventStream creates a new event stream for the session being
 // managed.
 func (manager *sessionManager) StartEventStream() {
-	// Start the event stream on a separate go routine
-	go manager.handleEvents()
+	manager.handleEvents()
```

Comment:
I assume this change is fine.  I didn't see any changes on manager.handleEvents() to start a new go routine, but I did see other places where the for loop was wrapped in a goroutine.

Reply:
You are correct. The go routine was moved to be around just the for loop.